flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/api/TableEnvironmentTest.java

```diff
@@ -110,4 +118,96 @@ public void testTableFromDescriptor() {
 
         assertEquals("fake", lookupResult.get().getTable().getOptions().get("connector"));
     }
+
+    @Test
+    public void testCreateManagedTableFromDescriptor() {
```

Comment:
I would prefer splitting this into separate test cases. If one case fails, we'll immediately know which one. Right now this is one giant test case that actually tests several cases, and several things per case.